.github/workflows/main.yml

```diff
@@ -24,15 +24,15 @@ jobs:
         if: startsWith(runner.os, 'Linux')
         with:
           path: ~/.cache/pip
-          key: ${{ runner.os }}-${{ matrix.python-version }}-pip-${{ hashFiles('**/*requirements.txt') }}
+          key: ${{ runner.os }}-${{ matrix.python-version }}-pip-${{ hashFiles('requirements/*requirements.txt') }}
           restore-keys: |
             ${{ runner.os }}-${{ matrix.python-version }}-pip-
 
       - name: Install
         run: |
           pip install --upgrade pip
-          pip install -r requirements.txt
-          pip install -r test_requirements.txt
+          pip install -r requirements/requirements.txt
```

Comment:
Did we want to rename `requirements.txt` to `full_requirements.txt` or something?

Reply:
Given that it is in fact not just the output of requirements.in anymore

Reply:
Agreed!